helpers/functions.py

```diff
@@ -601,3 +601,10 @@ def perpetuity_payment(present_value:float,rate:float):
     payment = present_value * (rate/100)
     return payment
 
+#function to calculate amortized loan payment 
+def amortized_loan_payment(amount_borrowed:float,interest_rate_per_period:float,number_of_payments:float):
+    a = amount_borrowed*interest_rate_per_period
```

Comment:
The number of payments is taken in as year but during the calculation, it is converted into months. Please look into this.